platform/locale_linux.cpp

```diff
@@ -4,7 +4,7 @@ namespace platform
 {
 Locale GetCurrentLocale()
 {
-  return {"", "", ""};
+  return {{}, {}, {}, ".", ","};
```

Comment:
Same here

Reply:
Same as before, I agree to set the empty "" grouping separator as default, though this change affects the unit testing code, so I will have to change back again the unit tests for distances.